lib/doggy/cli.rb

```diff
@@ -26,9 +26,10 @@ def pull(*ids)
       the marker in their title will get it as a result of a push.
     D
 
-    method_option "dashboards", type: :boolean, desc: 'Pull dashboards'
-    method_option "monitors",   type: :boolean, desc: 'Pull monitors'
-    method_option "screens",    type: :boolean, desc: 'Pull screens'
+    method_option "dashboards",   type: :boolean, default: true, desc: 'Pull dashboards'
+    method_option "monitors",     type: :boolean, default: true, desc: 'Pull monitors'
+    method_option "screens",      type: :boolean, default: true, desc: 'Pull screens'
+    method_option "all_objects",  type: :boolean, default: false, desc: 'Push all objects even if they are not changed'
```

Comment:
Indeed. This is potentially destructive. A simple confirmation message would confirm that people are doing what they intend.